ocaml/idl/datamodel.ml

```diff
@@ -7107,6 +7107,17 @@ let pool_remove_from_guest_agent_config = call
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+let pool_set_default_sr = call
+    ~name:"set_default_SR"
+    ~in_product_since:rel_ely
```

Comment:
This function has always been in the API, so it is wrong to say it is new in Ely.
Instead the lifecycle should show it as being `Published` in Rio, and I think it would be good to mention the `Changed` behaviour in Ely.